File: backend/src/main/java/com/backend/api/question/dto/response/AiQuestionReadAllResponse.java
package com.backend.api.question.dto.response;

import java.util.List;
import java.util.UUID;

public record AiQuestionReadAllResponse(
        List<AiQuestionReadResponse> questions
) {
    public static AiQuestionReadAllResponse from(List<AiQuestionReadResponse> responses){
        return new AiQuestionReadAllResponse(responses);
    }
}

File: backend/src/main/java/com/backend/api/question/dto/response/AiQuestionReadAllResponse.kt
package com.backend.api.question.dto.response


data class AiQuestionReadAllResponse(
    val questions: List<AiQuestionReadResponse>
) {
    companion object {
        fun from(responses: List<AiQuestionReadResponse>): AiQuestionReadAllResponse {
            return AiQuestionReadAllResponse(responses)
        }
    }
}